tests/features/tmgmt/tmgmt_poetry.feature

```diff
@@ -417,3 +417,44 @@ Feature: TMGMT Poetry features
       And I press "Request translation"
       And I press "Submit to translator"
       Then I see the date of the last change in the "French" row
+
+  @javascript
+  Scenario: Adding new languages to the ongoing translation request
+    Given I am logged in as a user with the 'editor' role
+    And I have the 'contributor' role in the 'Global editorial team' group
+    And I am viewing a multilingual "page" content:
+      | language | title            | body                    |
+      | en       | Title            | Last change column test |
+    When I click "Translate" in the "primary_tabs" region
+    Then I should not see "Request addition of new languages"
+    When I check the box on the "French" row
+    And I check the box on the "Portuguese" row
+    And I press "Request translation"
+    And I press "Submit to translator"
+    And I store the job reference of the translation request page
+    Then I should not see "Request addition of new languages"
+    And I should see "None" in the "German" row
+    And I should see "None" in the "Italian" row
+    When I am logged in as a user with the 'administrator' role
+    And I go to "admin/poetry_mock/dashboard"
+    And I click "Send 'ONG' status" in the "en->fr" row
+    And I click "Send 'ONG' status" in the "en->pt-pt" row
```

Comment:
A bit unfortunate that new steps are introduced that require clicks in the dashboard that a real user would not do.

The scenario is switching now to the administrator role and not back to editor, while we should test that an editor can request addition of new languages AFAIK.

Suggestion for improved step names (without using the dashboard):
_And DGT accepts the translation request with target language "fr"_
_And DGT accepts the translation request with target language "pt-pt"_